agent/roles/manifests/templates/appliance-config_yaml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1beta1
+kind: ApplianceConfig
+ocpRelease:
+  version: {{ version }}
+  channel: candidate
+  cpuArchitecture: x86_64
+diskSizeGB: 200
```

Comment:
Can the diskSizeGB be set to a smaller size? 200G per VM fills up the disk on my local test machine. What would be a recommended minimum size?